integration-tests/narayana-jta/src/test/java/io/quarkus/narayana/jta/TransactionScopedTest.java

```diff
@@ -0,0 +1,66 @@
+package io.quarkus.narayana.jta;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import javax.enterprise.context.*;
+import javax.inject.*;
+import javax.transaction.*;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTest
+class TransactionScopedTest {
+    @Inject
+    private UserTransaction tx;
+
+    @Inject
+    private TransactionManager tm;
+
+    @Inject
+    private TransactionScopedBean beanTransactional;
+
+    @Inject
+    private TransactionBeanWithEvents beanEvents;
+
+    @Test
+    public void transactionScopeInTransaction() throws Exception {
+        // TransactionScopedBean beanTransactional = Arc.container().instance(TransactionScopedBean.class).get();
```

Comment:
Just a reminder to delete this commented line later on.
I've yet to check why it didn't work for you...